hieradata/clients/azure.ci.jenkins.io.yaml

```diff
@@ -267,6 +267,13 @@ profile::jenkinscontroller::jcasc:
             cpus: 4
             memory: 8
             imagePullSecrets: dockerhub-credential
+          - name: jnlp-maven-8-all-in-one
+            labels:
+              - maven-8-all-in-one
+            cpus: 4
+            memory: 8
+            imagePullSecrets: dockerhub-credential
+            javahome: /opt/jdk-8
```

Comment:
```suggestion
            javaHome: /opt/jdk-8
```

Reply:
`            javaHome: /opt/jdk-8`
shouldn't it be javaPath to match the templating ? 
`value: "<%= agent['javaPath'] %>"`